.github/workflows/platform.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
     paths:
       - "rust-toolchain.toml"
+      - "rust-tools.txt"
```

Comment:
Move the rust-tools.txt to scripts/setup/?
So we can keep the root directory cleaned.